usr/share/regolith-look/blackhole/typeface

```diff
@@ -0,0 +1,48 @@
+! Typeface: Source Code Pro
+#define typeface_bar_glyph  Source Code Pro Medium 9
+#define typeface_wm         pango:Source Code Pro Medium 9
+#define typeface_bar        pango:Source Code Pro Medium 9, FontAwesome 9
+#define typeface_rofi       Source Code Pro Medium 16
+#define typeface_terminal   Source Code Pro:pixelsize=12:antialias=true:autohint=true
+#define typeface_gnome_terminal Source Code Pro 12
+
+#define typeface_bar_glyph_wifi 
```

Comment:
After you switch the font you'll notice that some of the glyphs are no longer correct because the underlying unicode values have changed.